web-admin/src/routes/+layout.svelte

```diff
@@ -49,9 +42,7 @@
 
 <RillTheme>
   <QueryClientProvider client={queryClient}>
-    <main
-      class="flex flex-col min-h-screen {onMetricsExplorerPage && 'h-screen'}"
-    >
+    <main class="flex flex-col min-h-screen h-screen">
```

Comment:
I like the idea to simplify this, but one regression is that pages w/ scroll no longer have a bottom padding

Before:
![image](https://github.com/rilldata/rill/assets/14206386/cf037ba6-75c1-4870-8217-a45ee07915da)

After:
![image](https://github.com/rilldata/rill/assets/14206386/827213d7-5d2a-441c-8d8d-fec725169281)

Reply:
Added bottom padding to the content container which seems to be used on all the routes that need it.